### ISSUE TITLE
reproduce: learning rate and loss weight

### ISSUE THREAD

**Issue by rheallyc** [zhangquanchen/3DThinker#6]
Hi, amazing work!
I wish to reproduce the results on Qwen2.5VL-3B-Instruct. But I have some confusions about experiment settings.
**1. loss weight**
The weight ratio in paper is 0.1 : 1 for 3D:text. In the `trainer_single.py`, the sim_loss is multiplied by 0.0005, and final ratio is 1 : 0.1 instead of 0.1 : 1
```
sim_loss = feature_sim.mean()*0.0005
loss = 0.1 * ce_loss + sim_loss
```
Also, the use of `model.projector_model` is different across `trainer_single.py`, `trainer_multi.py`, `trainer_deepspeed.py`
**2. lr setting.** 
May I know whether we should use 1e-4 or 1e-5? also different between `single` `multi` `deepspeed`.
**3. batch size.** 
May I know the batch size (considering gradient accumulation) used to train the best model? it's not discussed in paper
